hydrus/tests/test_crud.py

```diff
@@ -194,11 +214,13 @@ def test_insert_type(self):
             response_code, message = e.get_HTTP()
         assert 400 == response_code
 
-    def test_insert_id(self):
+    def test_insert_used_id(self):
         """Test CRUD insert when used ID is given."""
         object_ = gen_dummy_object(random.choice(
             self.doc_collection_classes), self.doc)
-        id_ = "1"
+        id_ = str(uuid.uuid4())
+        insert_response = crud.insert(

```

Comment:
Why this insert out of the try catch block?

Reply:
The first insert is a normal insert with a unique ID and the other one in the try block is redundant (using already used id, which is supposed to fail).

Reply:
I have also changed the name of this test to make it more clear.